presto-cli/src/main/java/io/prestosql/cli/ClientOptions.java

```diff
@@ -366,4 +379,66 @@ public boolean equals(Object obj)
                     Objects.equals(this.value, other.value);
         }
     }
+
+    public static final class ClientExtraCredentials
+    {
+        private final String name;
+        private final String value;
+
+        public ClientExtraCredentials(String extraCredentials)
+        {
+            List<String> nameValue = NAME_VALUE_SPLITTER.splitToList(extraCredentials);
+            checkArgument(nameValue.size() == 2, "Extra credential: %s", extraCredentials);
```

Comment:
Please update the message and say what is wrong with the extra credentials.

Reply:
I'd keep it in a consistent way as others in the `ClientOptions`.